Eplant/views/SettingsView/component.tsx

```diff
@@ -0,0 +1,39 @@
+import { useConfig } from '@eplant/config'
+import { useSettings } from '@eplant/state'
+import { ViewProps } from '@eplant/View'
+import { FormControlLabel, FormGroup, Switch, Typography } from '@mui/material'
+import React from 'react'
+import DebugView from '../DebugView'
+
+export default function SettingsView({
+  geneticElement,
+  activeData,
+}: ViewProps<any, any, any>) {
+  const [settings, setSettings] = useSettings()
+  return (
+    <>
+      <FormGroup>
+        <FormControlLabel
+          control={
+            <Switch
+              onChange={(e) => setSettings({ ...settings, darkMode: e.target.checked})}
+              checked={settings.darkMode}
+            />
+          }
+          label="Dark Mode"
+        />
+      </FormGroup>
+      <FormGroup>
+        <FormControlLabel
+          control={
+            <Switch
+              onChange={(e) => setSettings({...settings, debugMode: e.target.checked})}
+              checked={settings.debugMode}
+            />
+          }
+          label="Show Debug View"
```

Comment:
This should make it clear that the debug view will only be visible after reloading. 
```suggestion
          label="Show Debug View (requires reload)"
```